src/diff/index.js

```diff
@@ -352,7 +352,7 @@ export function unmount(vnode, ancestorComponent, skipRemove) {
 		}
 	}
 
-	if (dom!=null) removeNode(dom);
+	if (dom) removeNode(dom);
```

Comment:
We should test this with an empty textNode/textNode with 0. Not entirely sure if this would still work

Reply:
I'll try to dig into this later

Reply:
if `dom instanceof Text` it should work fine. IIRC the reason for using `!=null` and `==null` all over the codebase was performance + byte size. We'll want to check for perf regressions if we swap them, and make sure we use them uniformly.